src/khoj/routers/api_chat.py

```diff
@@ -583,6 +586,41 @@ async def send_rate_limit_message(message: str):
             await conversation_command_rate_limiter.update_and_check_if_valid(websocket, cmd)
             q = q.replace(f"/{cmd.value}", "").strip()
 
+        if conversation_commands == [ConversationCommand.Summarize]:
+            file_filters = conversation.file_filters
+            response_log = ""
+            if len(file_filters) == 0:
+                response_log = "No files selected for summarization. Please add files using the section on the left."
+                await send_complete_llm_response(response_log)
+            elif len(file_filters) > 1:
+                response_log = "Only one file can be selected for summarization."
+                await send_complete_llm_response(response_log)
+            else:
+                try:
+                    file_object = await FileObjectAdapters.async_get_file_objects_by_name(user, file_filters[0])
```

Comment:
Need to add error handling here in case the file_object isn't there. This would be the case for any file that's been previously uploaded.